web/client/components/mapviews/settings/CompactRichTextEditor.jsx

```diff
@@ -57,7 +57,7 @@ function CompactRichTextEditor({
                 image: {
                     urlEnabled: true,
                     // upload controlled via props, disabled by default
-                    uploadEnabled: props.uploadEnabled || false,
+                    uploadEnabled: props.enableUploadImg || false,
```

Comment:
are you sure that renaming this is correct?

is there any specific reason? if not please keep old prop name